uber/utils.py

```diff
@@ -548,6 +548,28 @@ def remove_opt(opts, other):
     return ','.join(map(str, set(opts).difference(other)))
 
 
+def get_age_from_birthday(birthdate, today=None):
+    """
+    Determines a person's age in the US. DO NOT use this to find other timespans, like the duration of an event.
+    This function does not calculate a fully accurate timespan between two datetimes.
+    This function assumes that a person's age begins at zero, and increments when `today.day == birthdate.day`.
+    This will not be accurate for cultures which calculate age differently than the US, such as Korea.
+
+    Args:
+        birthdate: A date, should be earlier than the second parameter
+        today:  Optional, age will be found as of this date
+
+    Returns: An integer indicating the age.
+
+    """
+
+    if today == None:
```

Comment:
You can skip these two lines by just using the parameter default instead, e.g., `def get_age_from_birthday(birthdate, today=localized_now()):`

Reply:
I read that this would cause the default value to be evaluated when the function is defined, rather than when it's called. Couldn't that potentially cause undesireable behavior if a system is running for a long period of time? "Now" would gradually become outdated, forever pointing to the day the function was defined. (is this when the server was started?)

Or am I misunderstanding how this works?

Reply:
Oh gosh, you're totally right! I think I knew that once and then forgot, my bad!